src/test/java/PlaylistTest.java

```diff
@@ -0,0 +1,65 @@
+import Pages.HomePage;
+import Pages.LoginPage;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PlaylistTest extends BaseTest {
+
+    @Test(priority = 0)
+    public void createNewPlaylistTest(){
+        LoginPage loginPage = new LoginPage(getDriver());
+        HomePage homePage = new HomePage(getDriver());
+        loginPage.login();
+        homePage.createPlaylist("New Test Playlist");
+        //Assertions
+        Assert.assertTrue(homePage.isPlaylistCreatedMessageDisplayed());
+    }
+
+
+    /*
+    * Rename Existing Playlist, there is a problem that failed with the same name
+    * */
+//    @Test(priority = 1)
+//    public void renameExistPlaylist() {
+//        LoginPage loginPage = new LoginPage(driver);
+//        HomePage homePage = new HomePage(driver);
+//        BasePage basePage = new BasePage(driver);
+//        String updatePlaylistMsg = "Updated playlist \"Sample Edited Playlist.\"";
+//        String newPlaylistName = "Sample Edited Playlist";
+//
+//        loginPage.login();
+//        basePage.doubleClickPlaylistElement(By.cssSelector(".playlist:nth-child(4)"));
+//        homePage.enterNewPlaylistName(newPlaylistName, By.cssSelector("[name='name']"));
+//        //Assertions
+//        Assert.assertEquals(homePage.getRenamePlaylistSuccessMsg(By.cssSelector("div.success.show")), updatePlaylistMsg);
+//    }
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.